### PR TITLE
Menú Responsive 

### PR DESCRIPTION
**Pull request by condef5**
Se ha agregado responsive desing al menú.

### CODE REVIEW

File: app/controllers/links_controller.rb [hunk 1]
```diff
@@ -71,4 +71,4 @@ def set_link
     def link_params
       params.require(:link).permit(:title, :url)
     end
-end
+  end
```
Comment (moisesnandres) [inline app/controllers/links_controller.rb]: the final end shouldn't be indented

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -4,11 +4,10 @@
 	</h2>
 	<ul class="lists">
 		<% @links.each do |link| %>
-			<li>
-				<a href="<%= link.url %>" target="_blank" class="link"><%= link.title %></a>
-			</li>
+		<li>
```
Comment (moisesnandres) [inline app/views/home/index.html.erb]: now indent your code to 2 spaces, there are 4 spaces

File: app/assets/stylesheets/home.scss [hunk 3]
```diff
@@ -12,6 +12,14 @@ body{
 	margin: 50px 0 30px;
 	font-size: 40px;
 }
+@media screen and (max-width: 400px) {
```
Comment (SofiaChuquin) [inline app/assets/stylesheets/home.scss]: All your files must be indented with 2 spaces.

File: app/assets/stylesheets/home.scss [hunk 4]
```diff
@@ -12,6 +12,14 @@ body{
...
+	.content .is-title-1{
+		font-size: 30px;
+		margin: 0;
+		padding-left: 10px;
+		text-align: left;
+	}
+}
```
Comment (SofiaChuquin) [inline app/assets/stylesheets/home.scss]: All the files must be indented with 2 spaces.

**Pull request merged**